app/src/main/java/edu/uc/hickmadc/customnotifications/MainActivity.kt

```diff
@@ -1,38 +1,36 @@
 package edu.uc.hickmadc.customnotifications
 
-import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import android.widget.ListView
+import androidx.appcompat.app.AppCompatActivity
 import androidx.navigation.Navigation
 import androidx.navigation.findNavController
 import androidx.navigation.ui.NavigationUI
 import com.google.android.material.bottomnavigation.BottomNavigationView
 import edu.uc.hickmadc.customnotifications.ui.main.MyListAdapter
 
-//import edu.uc.hickmadc.customnotifications.ui.main.MainFragment
```

Comment:
Don't comment out imports, it makes code look dirty.

Reply:
Agreed.   Committing commented out code is considered bad practice in version control, because version control offers better alternatives:
1) Use a feature branch for experimental features.
2) Delete what is not needed.  You can always compare with history to see what used to be there.
Commented out code is considered clutter in a version control system.